.env.sample

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=dev-jz8ljx4f.us.auth0.com
```

Comment:
These shouldn't be in `.env.sample`. You'll want to recycle those addresses/keys. Put a random string for these instead of the actual keys/secrets.

Reply:
So put random strings and each person will be given the correct sercet once they need to add into the env file.

Reply:
Correct, or you can leave it blank after the environment variables themselves - either way, these are things we'll want to add to our `.env` documentation and make sure everyone adds to their `.env` files! I'm adding this to our docs now.